### PR TITLE
added video upload feature 

### PR DESCRIPTION
**Pull request by rogueloop**
Add videos in this format in a single line:

`<video width="600" controls>  <source src="./createDocs.mp4" type="video/mp4"></video>`

### CODE REVIEW

File: uploadBlogVideos.go [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+		log.Info("Videos Upload videoData...",videoData)
+		if err != nil {
+			log.Error("Error calculating hash:", err)
+			return nil, err
+		}
+
+		var videoLink string
+		if strings.HasPrefix(video, "http://") || strings.HasPrefix(video, "https://") {
+			if videoBackend == "s3" {
+				log.Info("Uploading video o S3...",videoLink,videoData,hashValue)
```
Comment (shrsv) [inline uploadBlogVideos.go:52]: If it's an externally hosted video (not a file reference - relative/absolute) - do we need to upload?

Comment (rogueloop) [reply uploadBlogVideos.go:52]: the code is there, but haven't tried and verified

Comment (shrsv) [reply uploadBlogVideos.go:52]: I think we don't have to deal with this case - we should do nothing. Only if it is a local file - we may upload.

File: uploadBlogVideos.go [hunk 2]
```diff
@@ -0,0 +1,57 @@
+package main
+
+import (
+	"regexp"
+)
+
+func uploadVideos(token, htmlData string) (map[string]string, error) {
+	uploadedVideos := make(map[string]string)
+	videoBackend := config.GetDefault("image-configuration.VIDEO_BACKEND", "").(string)
```
Comment (LinceMathew) [inline uploadBlogVideos.go:9]: is this supposed to be `video-configuration.VIDEO_BACKEND`?

**Pull request merged**